code/__DEFINES/actions.dm

```diff
@@ -8,13 +8,16 @@
 #define AB_CHECK_CONSCIOUS (1<<3)
 ///Action button checks if user is incapacitated
 #define AB_CHECK_INCAPACITATED (1<<4)
+///Action button checks if user is jaunting
+#define AB_CHECK_PHASED (1<<5)
```

Comment:
I almost feel like this should replace `SPELL_CASTABLE_WHILE_PHASED` req on spell actions